app/controllers/projects_controller.rb

```diff
@@ -116,7 +116,7 @@ def new_user_for_project
   end
 
   def add_user_to_project
-    user = User.find(params[:project][:user])
+    user = current_company.users.find(params[:project][:user])
```

Comment:
What is the reasoning behind using `find` at one place and `find_by_*` at the other place?